lib/netuitived/metric_aggregator.rb

```diff
@@ -7,6 +7,7 @@ class MetricAggregator
     def initialize(apiEmissary)
       @metrics = []
       @samples = []
+      @tags = []
```

Comment:
This could be a local variable down in line 58 if you'd like to collapse the logic to that section only.

Reply:
Agreed. I'll move it down.